platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBookValidator.java

```diff
@@ -92,6 +94,61 @@ public static boolean validNextId(final AddressBook previousAddressBook, final A
         return true;
     }
 
+    /**
+     * Validate that the nextNodeId in the new address book is greater than or equal to the nextNodeId in the old
```

Comment:
This sentence is a lot to take in. Can it be split into bullets?

Reply:
I split it into bullets.